tests_metricflow/query_rendering/test_granularity_date_part_rendering.py

```diff
@@ -437,3 +439,53 @@ def test_subdaily_granularity_overrides_metric_default_granularity(  # noqa: D10
         dataflow_plan_builder=dataflow_plan_builder,
         query_spec=query_spec,
     )
+
+
+@pytest.mark.sql_engine_snapshot
+def test_date_part_with_non_default_grain(  # noqa: D103
+    request: FixtureRequest,
+    mf_test_configuration: MetricFlowTestConfiguration,
+    dataflow_plan_builder: DataflowPlanBuilder,
+    dataflow_to_sql_converter: DataflowToSqlQueryPlanConverter,
+    sql_client: SqlClient,
+    query_parser: MetricFlowQueryParser,
+) -> None:
+    query_spec = query_parser.parse_and_validate_query(
+        metric_names=["archived_users"],
+        group_by=(
+            TimeDimensionParameter(name="metric_time", grain=TimeGranularity.HOUR.value, date_part=DatePart.YEAR),
+        ),
+        where_constraint_strs="{{ TimeDimension('metric_time', 'month', date_part_name='day') = '2020-01-01' }}",
```

Comment:
need to wrap this in a list, otherwise you get https://github.com/dbt-labs/metricflow/pull/1599/commits/dad3755d1754d2604a048034858b6b1a4c67731d#diff-5d8fe0debedfef559a1f628e0cc669d322f1a1d1484ad11706e3980fe78b1b8bR20 😆

Reply:
Oooops thank you!!